book/custom_commands.md

```diff
@@ -219,31 +404,38 @@ def greet [
   name: string
   --age: int
 ] {
-  [$name $age]
-}
+    {
+      name: $name
+      age: $age
+    }
+  }
```

Comment:
This last line is indented a bit, which seems to be inconsistent with future definitions (also present in the next definition of `greet` below which demonstrates shorthand flags)

Reply:
Thanks - Thought I'd caught all of them, but missed that one.  Completed in the upcoming commit.